otherlibs/systhreads/st_stubs.c

```diff
@@ -426,15 +441,14 @@ CAMLprim value caml_thread_cleanup(value unit)
 
   return Val_unit;
 }
-/* Thread cleanup at termination */
 
 static void caml_thread_stop(void)
 {
   /* PR#5188, PR#7220: some of the global runtime state may have
      changed as the thread was running, so we save it in the
-     curr_thread data to make sure that the cleanup logic
+     this_thread data to make sure that the cleanup logic
```

Comment:
is `this_thread` an old name for `Active_thread`?

Reply:
This is a rebasing error, which will disappear with the next rebasing :)